production/helm/loki/templates/admin-api/deployment-admin-api.yaml

```
@@ -97,14 +97,8 @@ spec:
             - name: http-memberlist
               containerPort: 7946
               protocol: TCP
-          {{- with .Values.adminApi.livenessProbe }}
-          livenessProbe:
-            {{- toYaml . | nindent 12 }}
-          {{- end }}
-          {{- with .Values.adminApi.readinessProbe }}
-          readinessProbe:
-            {{- toYaml . | nindent 12 }}
-          {{- end }}
+          {{- include "loki.adminApi.readinessProbe" . | nindent 10 }}
+          {{- include "loki.adminApi.livenessProbe" . | nindent 10 }}
```

Hi, so please keep the probe configuration inside the templates. Having a template for a one time usage make thinks more complicated.
```suggestion
          {{- with .Values.adminApi.livenessProbe | default .Values.loki.livenessProbe }}
          livenessProbe:
            {{- toYaml . | nindent 12 }}
          {{- end }}
          {{- with .Values.adminApi.readinessProbe | default .Values.loki.readinessProbe }}
          readinessProbe:
            {{- toYaml . | nindent 12 }}
          {{- end }}
```